packages/next/build/swc/index.js

```diff
@@ -0,0 +1,71 @@
+const { loadBinding } = require('@node-rs/helper')
+const path = require('path')
+
+/**
+ * __dirname means load native addon from current dir
+ * 'next-swc' is the name of native addon
+ * the second arguments was decided by `napi.name` field in `package.json`
+ * the third arguments was decided by `name` field in `package.json`
+ * `loadBinding` helper will load `next-swc.[PLATFORM].node` from `__dirname` first
+ * If failed to load addon, it will fallback to load from `next-swc-[PLATFORM]`
+ */
+const bindings = loadBinding(
+  path.join(__dirname, '../..'),
```

Comment:
Currently the next-swc binaries will output to the base of the next package. We will have to decide how to publish them eventually, but for now I wanted to keep them somewhere that won't get published until next-swc-loader is functional behind a flag.  Where would be a good place to output them for now?

Reply:
Decided to output to `packages/next/native`